Makefile

```
@@ -393,3 +393,14 @@ kind-integration-test: .local/kind-kubeconfig
 
 kind-integration-e2e: .local/kind-kubeconfig
 	cd services/brig && KUBECONFIG=$(CURDIR)/.local/kind-kubeconfig ./federation-tests.sh $(NAMESPACE)
+
+kind-restart-all: .local/kind-kubeconfig
+	export KUBECONFIG=$(CURDIR)/.local/kind-kubeconfig
```

Exporting an environment variable on one line of Make and expecting it to be available in the next one doesn't work. Workaround: Use `&&` and a newline escape.
```suggestion
	export KUBECONFIG=$(CURDIR)/.local/kind-kubeconfig && \
```

---

Fixed it, thanks!